fala/apps/adviser/migrations/0002_alter_feedback_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.6 on 2024-05-24 12:47
```

Comment:
While having any number of migrations is fine and once we're under way there's no need to be thrifty with them (as stated explicitly in the docs themselves) I would recommend it's good practice at this point to combine this and the above migration. If nothing else, it's good exercise.

Your options are likely:
- delete both migrations and run `makemigrations` again
- use `squashmigrations` ([docs here](https://docs.djangoproject.com/en/5.0/topics/migrations/#squashing-migrations))
- manually edit the first migration

Although each of these, possibly apart from the second, probably involves blowing away your local db first as it keeps track of what migrations have been applied and deleting migrations would impact that. If you can migrate back to _before_ the initial migration (I know back in the day you couldn't but that may have changed) then you could do that, then delete the migrations, then make new ones, then migrate.

So yeah, maybe it's not worth it at all 😂

Reply:
Thanks, I understand now what happened, the original migration failed and so I added the BigAutoField and re-ran the migration, thinking I would get one successful migration. I will do as you suggest and blow it away and then rerun them as a single migration.